### PR TITLE
new changes

### PR DESCRIPTION
**Pull request by YafeiXu**

### CODE REVIEW

File: 809distribution_binom/MMSTATdistribution_binom.r [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # ------------------------------------------------------------------------------
-# Book:         MMStat
```
Comment (ebommes) [inline 809distribution_binom/MMSTATdistribution_binom.r]: Why did you change the project name?
http://quantlet.de/index.php?p=searchResults&w=project&id=452

File: 809distribution_binom/MMSTATdistribution_binom.r [hunk 2]
```diff
@@ -18,21 +18,24 @@
 # ------------------------------------------------------------------------------
 # output:       Interactive shiny application
 # ------------------------------------------------------------------------------
-# Example:      The given application example shows a histogram of the binomial distribution with number of draws
```
Comment (ebommes) [inline 809distribution_binom/MMSTATdistribution_binom.r]: Please see Lukas commit regarding formatting of the header (in his MMSTAT commit he stated that the ~ 80 character rule does not apply to the header)

File: 809distribution_binom/MMSTATdistribution_binom.r (hunk 1 above)
Comment (YafeiXu) [inline 809distribution_binom/MMSTATdistribution_binom.r]: I see all Qs have been changed with the new title. see the following link.

http://quantnet.wiwi.hu-berlin.de/index.php?p=show&id=3363

Best Regards,
Yafei Xu

2015-08-11 16:04 GMT+02:00 ebommes <notifications@github.com>:

File: 809distribution_binom/MMSTATdistribution_binom.r [hunk 3]
```diff
@@ -18,21 +18,24 @@
 # ------------------------------------------------------------------------------
 # output:       Interactive shiny application
 # ------------------------------------------------------------------------------
-# Example:      The given application example shows a histogram of the binomial distribution with number of draws
-#               and probability of success per draw chosen from slide-bars. One can see the bins (in gray) to 
-#               approximate the PDF of the distribution. 
+# Example:      The given application example shows a histogram of the binomial
```
Comment (ebommes) [inline 809distribution_binom/MMSTATdistribution_binom.r]: Please think about whether the English in the Example section is correct.
E.g. (not i.e. !!)  maybe the bins (in gray) that approximate the PDF.

Comment (YafeiXu) [inline 809distribution_binom/MMSTATdistribution_binom.r]: ok according to your suggestion, new change has been made. are they ok now?

Best Regards,
Yafei Xu

2015-08-11 16:10 GMT+02:00 ebommes <notifications@github.com>:
